### ISSUE TITLE
Talented Spells not Working - Warrior

### ISSUE THREAD

**Issue by Rusher1337** [ps-wow/Broker_Classic_Training#21]
['mortal_strike:2'] = {
      name = 'Mortal Strike',
      rank = 2,
      id = 21551,
      level = 48,
      cost = 200,
      replaced_by = { 21552, 21553 },
      talent = 'mortal_strike:1'
    },

etc. are not working, Mortal Strike is not talented and the higher ranks are shown as unlearned.

**Comment by davidcraig** [comment]
Yes I plan on doing talents differently, just haven't gotten around to them, my thinking was to only check the later ones if the earlier one is trained, similar to how replaced_by works.
